plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -127,7 +127,20 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	}
 
 	trace.Step("Selecting host")
-	return g.selectHost(priorityList)
+	host, err := g.selectHost(priorityList)
+	if err != nil {
+		return "", err
+	}
+
+	if len(g.extenders) != 0 {
+		for _, extender := range g.extenders {
+			err := extender.Bind(pod, host)
```

Comment:
> How do you handle bind failures now, before this change?

Currently, it will mark task as FAILED if bind apiserver failed.
In kube-mesos, it'll let tasks to do apiserver bind, so: if failed to create task, not bind; if failed to bind, task will mark as FAILED. No cache in kube-mesos.